scope.Rmd

```diff
@@ -110,62 +104,155 @@ context of statistical software.
   figure reflects general code in all R packages, and the corresponding
   proportion within the specific context of statistical software may be
   considerably lower.
-
-***Proposal***: Peer review in the system will primarily focus on code written
-in R, C, and C++. Standards will be written so as to separate language-specific
-and non-language-specific components with an eye towards further adoption by
-other groups in the future (in particular groups focussed on the Python
-language). Any decision to include visualisation software and routines within
-our scope will likely entail an extension of linguistic scope to associated
-languages (HTML, JavaScript, and maybe CSS).
+- Any decision to include visualisation software and routines within our scope
+  will likely entail an extension of linguistic scope to associated languages
+  (HTML, JavaScript, and maybe CSS).
 
 ### Structure
 
-R has a [well-defined system](https://cran.r-project.org/doc/manuals/R-exts.html) for structuring software packages"
-Other forms of packaging **R** software may nevertheless be considered within scope. These may include
+R has a [well-defined
+system](https://cran.r-project.org/doc/manuals/R-exts.html) for structuring
+software packages" Other forms of packaging **R** software may nevertheless be
+considered within scope. These may include
 
-1. Python-like systems of
-[modules for **R**](https://github.com/klmr/modules).
-2. Scripts, such as standalone comand-line scripts
-3. Web applications such as Shiny packages
+1. Python-like systems of [modules for **R**](https://github.com/klmr/modules);
+2. Packaging appropriate for other languages (such as Python) yet with some
+   interface with the R language;
+3. R interfaces ("wrappers") to algorithms or software developed independently
+   in different languages, and which may or may not be bundled as a standard
+   R package; and
+4. Web applications such as Shiny packages.
 
 **Key considerations**: Allowing non-package forms of code into the peer review
```

Comment:
Besides, isn't there a trend to package all the things, Shiny app included? (see `golem`)